imports/ui/case-wizard/case-wizard.jsx

```diff
@@ -45,9 +45,7 @@ class CaseWizard extends Component {
         },
         optional: {
           category: null,
-          subCategory: null,
-          priority: null,
-          severity: null
+          subCategory: null
```

Comment:
@sophieyang0813 why do you need to remove line 49 and 50? Is the MEFE broken if this stays?

Reply:
@nbiton is it okay to change/indicate default values from the client side, even though priority/severity state wouldn't be changed from the user input? I tried to follow your suggestion on staying away from the client side so to avoid user manipulating input value.

Reply:
It's actually better to remove these lines as done in this commit. We don't need to have left overs of previous features lying around. It's just more confusing later on if it stays.